packages/core/src/components/status-label/status-label.stories.js

```diff
@@ -1,4 +1,8 @@
 import './status-label.css';
+/**
+ * Note: default rectangular StatusLabel will be deprecated in 3.0.0. 
+ * Rounded corder variant will be used as default instead.
+ */
```

Comment:
This is only informative to us developers I think but we could inform Storybook users about the deprecation with a notification component, similar to the doc site. I think it would actually be good to add that notification to the other deprecating components as well, would you mind doing that in this PR? The other deprecating or party deprecating are Tag (rectangular), Navigation and Footer.